### ISSUE TITLE
Archive chats (keep history)

### ISSUE THREAD

**Issue by mvaneijgen** [WebDevStudios/Chat-Room#37]
Maybe I am missing a setting, but when the page is reloaded only several of the messages are still there and the rest is gone.

Is there a way to keep the chat rooms history for later reference ?

**Comment by tw2113** [comment]
Yeah, we save, by day, the logs to a text file, it's not the most robust archive ability for the topic, but it is what it is. Honestly not the best tech for this type of thing, but it's all from before I was first hired too.
